hls-plugin-api/src/Ide/PluginUtils.hs

```diff
@@ -316,3 +319,10 @@ escapedTextParser = concat <$> P.many (outsideStringLiteral P.<|> stringLiteral)
           inside' = concatMap f inside
 
       pure $ "\"" <> inside' <> "\""
+
+-- ---------------------------------------------------------------------
+
+toAbsolute :: FilePath -> FilePath -> FilePath
+toAbsolute dir file
+    | isAbsolute file = file
+    | otherwise = dir </> file
```

Comment:
If I am not mistaken, this is equivalent to `toAbsolute = (</>)`, see https://hackage.haskell.org/package/filepath-1.5.2.0/docs/System-FilePath.html#v:-60--47--62-

Reply:
oh right, make it `toAbsolute = (</>)` now, but keep alias `toAbsolute` since it state our intension better

Reply:
done